ParkingGarageRentals/Program.cs

```diff
@@ -31,23 +31,37 @@ static void Main(string[] args)
                 Console.WriteLine("Press 9 to exit...");
                 Console.WriteLine("DailyPass: $5 (1), WeeklyPass: $15 (2), MonthlyPass: $30 (3), YearlyPass: $90 (4)");
                 Console.Write("Please choose a parking pass (Enter number 1 - 4): ");
-
-                try
-                {
-                    passNumberSelected = Convert.ToInt32(Console.ReadLine());
-                }
-                catch (FormatException)
+                do
                 {
-                    Console.WriteLine("Must enter a valid number.");
-                }
+                    if (passNumberSelected == 9)
+                    {
+                        isNotSentinelValue = false;
+                    }
```

Comment:
- [ ] Here's a place we can break out of the loop after setting `isNotSentinelValue = false;`
- [ ] Wrap lines 86-eom if-statement on `isNotSentinelValue` (if user presses 9, it should just exit this menu; no more processing in this method needs to occur)